addon/rollup.config.mjs

```diff
@@ -14,7 +15,7 @@ export default {
   plugins: [
     // These are the modules that users should be able to import from your
     // addon. Anything not listed here may get optimized away.
-    addon.publicEntrypoints(['modifiers/**/*.js', 'services/**/*.js', 'test-support/**/*.js']),
+    addon.publicEntrypoints(['**/*.js']),
```

Comment:
generally better sourcemap output occurs when publicEntrypoints is all files.

I have a task here: https://github.com/embroider-build/embroider/issues/1329
that would be a breaking change for `@embroider/addon-dev`, but it will make managing package.json#exports much easier, and optimize sourcemap generation as well as not pre-emptively optimize assets.

Rollup's default behaviors are meant for more production-ish builds, and since addons don't emit production-ready assets (we want the consuming app to make all those decisions), we want to be a bit lazier here for the dist-output. 
This strategy also makes patching and editing `node_modules` quite a bit nicer, if someone needs to do that.